source/frontend/src/features/mapSideBar/tabs/Section.tsx

```diff
@@ -20,7 +20,7 @@ export const Section: React.FC<
     <StyledFormSection className={clsx('form-section', className)}>
       {header && (
         <StyledSectionHeader>
-          <Row>
+          <Row className="no-gutters">
```

Comment:
Seems unlikely but just in case, can you notify Sue of this change? I feel like I've seen changes like this introduce scroll bars unexpectedly, but that may not be a problem here.